wpilibj/src/athena/java/edu/wpi/first/wpilibj/Relay.java

```diff
@@ -86,7 +86,7 @@ public InvalidValueException(String message) {
   private void initRelay() {
     SensorBase.checkRelayChannel(m_channel);
 
-    int portHandle = RelayJNI.getPort((byte)m_channel);
+    int portHandle = RelayJNI.getPort((byte) m_channel);
```

Comment:
checkstyle hasn't been reliable in my experience. All the Java formatters we've tried so far are just as bad.